tests/e2e/fixture/local/README.md

```diff
@@ -0,0 +1,201 @@
+
+# Start a new network
+$ ./build/avctl start-network --avalanchego-path=/path/to/avalanchego
+...
+Started network 1337 @ /home/me/.avctl/1337-20230705-213938-1707053131
+
+Configure avctl to target this network by default with one of the following statements:
+ - source /home/me/.avctl/1337-20230705-213938-1707053131/network.env
+ - export AVCTL_NETWORK_DIR=/home/me/.avctl/1337-20230705-213938-1707053131
+
+# Stop the network
+$ ./build/avctl stop-network --network-dir=/path/to/network
+```
+
+### Via code
+
+A local network can be managed in code:
+
+```golang
+network, _ := local.StartNetwork(
+    ginkgo.GinkgoWriter,
+    "", // Use default root dir (~/.avctl)
+    &local.LocalNetwork{
+        LocalConfig: local.LocalConfig{
+            ExecPath: "/path/to/avalanchego",  // Defining the avalanchego exec path is required
+        },
+	},
+    uint32(0), // Network id will be generated
+	5, // Nunber of initial validating nodes
+)
+
+uris := network.GetURIs()
+
+// Use URIs to interact with the network
+
+// Stop all nodes in the network
+network.Stop()
+```
+
+If non-default node behavior is required, the `LocalNetwork` instance
+supplied to `StartNetwork()` can be initialized with explicit node
+configuration and by supplying a nodeCount argument of `0`:
+
+```golang
+network, _ := local.StartNetwork(
+    ginkgo.GinkgoWriter,
+    "", // Use default root dir (~/.avctl)
+    &local.LocalNetwork{
+        LocalConfig: local.LocalConfig{
+            ExecPath: "/path/to/avalanchego",  // Default avalanchego path
+        },
+        Nodes: []*LocalNode{{
+            ExecPath: "/different/path/to/avalanchego", // Node-specific avalanchego path
+            Flags: FlagsMap{                            // Any and all node flags can be configured here
+                cfg.DataDirKey: "/custom/path/to/node/data",
+            },
+        },{ // node2 uses default configuration
+        },{ // node3 uses default configuration
+        },{ // node4 uses default configuration
+        },{ // node5 uses default configuration
+        }},
+	},
+    uint32(0), // Network id will be generated
+	0, // Number of nodes must be zero if explicitly setting node config
+)
+```
+
+Further examples of use code-based usage are located in the [e2e
+tests](../../e2e.go).
+
+## Networking configuration
+
+By default, nodes in a local network will be started with staking and
+API ports set to `0` to ensure that a port will be dynamically
+chosen. This currently requires watching log output of a node and
+parsing the structured log entries that detail the bound addresses of
+the API and staking ports. The use of dynamic ports supports testing
+with many local networks without regard to port availability.
+
+## Configuration on disk
+
+A local network relies on configuration written to disk in the following structure:
+
+```
+HOME
+└── .avctl                                           // Default parent directory for local networks
```

Comment:
Would suggest adding a `runs` subdirectory in case you ever needed to store additional types of files here